pkgs/development/interpreters/clojure/babashka.nix

```diff
@@ -31,7 +31,6 @@ stdenv.mkDerivation rec {
       "-H:IncludeResources=BABASHKA_VERSION" \
       "-H:IncludeResources=SCI_VERSION" \
       -H:ReflectionConfigurationFiles=${reflectionJson} \
-      --initialize-at-run-time=java.lang.Math\$RandomNumberGeneratorHolder \
```

Comment:
this is a little opaque - could you maybe add a comment to the commit message explaining why this isn't necessary anymore?

Reply:
Honestly, I have no idea. GraalVM keeps changing and a program that used to compile with one version of GraalVM and one set of flags suddenly doesn't in the next version, it's a mess.

Reply:
I can explain. This workaround was needed before because something in the java.lang.Math class was initialized at compile time, preventing true randomness. You would get the same random number over and over. This flag used to fixed it. But now this fix is built into graalvm 20.3.0 so the workaround must not be used anymore (due to how the fixed it, not sure about the details, but I think it results into a conflicting config).

Reply:
Thanks for the clarification @borkdude!